tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py

```diff
@@ -47,162 +48,254 @@
 references:
     - https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/index.html
     - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/vuln-detector.html#min_full_scan_interval
+    - https://github.com/wazuh/wazuh/issues/7749
+    - https://github.com/wazuh/wazuh/issues/8727
 
 tags:
     - settings
+    - general_settings
 '''
 import os
 import time
-from datetime import datetime, timedelta
-
 import pytest
 
-from wazuh_testing.modules import vulnerability_detector as vd
-from wazuh_testing import logger
-from wazuh_testing.tools import LOG_FILE_PATH, WAZUH_PATH
-from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
-from wazuh_testing.tools.monitoring import FileMonitor
-from wazuh_testing.tools.time import TimeMachine
+import wazuh_testing.vulnerability_detector as vd
+import wazuh_testing.db_interface.agent_db as adb
+from wazuh_testing import DB_PATH
 from wazuh_testing.tools.time import time_to_seconds
+from wazuh_testing.db_interface.cve_db import get_sqlite_query_result
+from wazuh_testing.modules.vulnerability_detector import event_monitor as evm
+from wazuh_testing.modules.vulnerability_detector.utils import clean_vd_tables
+from wazuh_testing.tools import configuration
```

Comment:
Sort the imports correctly, pls

Reply:
Solved https://github.com/wazuh/wazuh-qa/commit/6a6cfc73a5c0e11e4d3a086dd818cb61703ca4a8